.github/workflows/test-integration.yaml

```diff
@@ -43,6 +43,16 @@ jobs:
         run: ./hack/patch-hosts.sh
       - name: Integration Test
         run: make test-integration
+      - name: Dump Cluster Logs
+        if: always()
```

Comment:
This also could have been `failure()` but I think it shoudln't hurt having `always()`.